stdlib/jdbc/src/main/ballerina/src/java.jdbc/client.bal

```diff
@@ -102,6 +126,11 @@ public type Client client object {
     # Stops the JDBC client.
     #
     # + return - Possible error during closing the client
+    #
+    # # Deprecated
+    # New experimental jdbc2 module is introduced in 1.2 release and it will be fully supported with in 1.3 release
```

Comment:
```suggestion
    # A new experimentalJDBC2 module is introduced in the Ballerina 1.2 release and it will be fully supported by the 1.3 release.
```